### ISSUE TITLE
Add config file for customizing the layout

### ISSUE THREAD

**Issue by ghost** [cjbassi/ytop#39]
As i've mentioned in the previous issue i would be nice if displaying the graph is just an optional feature.

The reason i picked ytop for my defaul taks manager is because of its simpliciy compared to top, htop and glances.

**Comment by cjbassi** [comment]
Do you mean just the CPU and memory graphs? So I see a couple ways of implementing this:

1. have an option to toggle whether to show the graphs
2. add a config file to specify which widgets to show

The second option would be more work but would also be more general

**Comment by ghost** [comment]
I think both options are vaild

A toggle to switch on and off the graps and a modular approach for the rest of the components.

For example ytop -c (CPU) -m (memory) -t (temperature) -g (graphs). Would be the ideal set up for me. Only the information I want to see in a space without clutter and any other distractions.

**Comment by cjbassi** [comment]
I don't think specifying the widgets on the command line would work, unless we have users add the layout too, but that might get a little convoluted. So I think the best way to do this is just to have a config file where users specify which widgets they want and the layout for them.

**Comment by ghost** [comment]
I agree with you.

**Comment by cjbassi** [comment]
https://asciinema.org/a/BxIdMzYH3LtAmGCkabTFKsqbH?speed=2
https://github.com/cjbassi/gotop/pull/167
https://github.com/cjbassi/gotop/issues/111

**Comment by ghost** [comment]
What about the graphs ?

**Comment by cjbassi** [comment]
You should be able to remove the graphs using the config file. Does that work for you or were you thinking something else?

**Comment by ghost** [comment]
Well I mentioned at the start of the topic the reason i enjoy using ytop is because of its simplicity. The issue with the graphs is that they add distortion to that. I was thinking that by your mentioning a configuration file there would be the possibility to disable them there.

**Comment by cjbassi** [comment]
Yeah exactly, you should be able to disable them in the config file.

**Comment by ghost** [comment]
Thanks

**Comment by cjbassi** [comment]
You're welcome, but I'm going to reopen this since it still needs to be implemented :)

**Comment by ghost** [comment]
is it implemented in gotop ? because i dont have the config file present

**Comment by cjbassi** [comment]
No, it's not implemented in gotop. But there is a fork of gotop that has a config file implementation and I linked it above.
